api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -328,6 +328,11 @@ message HttpConnectionManager {
     // Defaults to false.
     bool cert = 3;
 
+    // Whether to forward the entire client cert chain in URL encoded PEM format. This will appear
```

Comment:
To avoid any confusion, could you note that it includes leaf certificate? Thanks!